README.md

```diff
@@ -15,11 +15,13 @@
 
 ## Configs
 
+- [stylelint-recommended](https://github.com/stylelint/stylelint-config-recommended) and [stylelint-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss) - The recommended shareable config for Stylelint.
+- [stylelint-standard](https://github.com/stylelint/stylelint-config-standard) and [stylelint-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss) - The standard shareable config for Stylelint (extends the recommended one).
+- [Prettier](https://github.com/prettier/stylelint-config-prettier) - Turns off all rules that are unnecessary when using Prettier.
```

Comment:
[suggest] How about adding also [`stylelint-config-prettier-scss`](https://github.com/prettier/stylelint-config-prettier-scss)?

Reply:
Ah yes of course! Thanks for that. Title updated also.